### PR TITLE
[zh]update verify-signed-images.md

### PR DESCRIPTION
**Pull request by ihuibin**
Previous PR #33956

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
+-->
+## 验证镜像签名 {#verifying-image-signatures}
+
+完整的镜像签名列表请参见[发行版本](/releases/download/)。
+
+我们从这个列表中选择一个镜像，并使用 `cosign verify` 命令来验证它的签名：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:55]: 比较好的做法是将上面这一段翻译完之后，直接保留 41-43 行的代码段不译，接下来在 `{{< note >}}` 块中分段注释、分段翻译。
像这样大段大段地注释和翻译比较难评审，并且代码段也被不必要地重复了。

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+example: `registry-url/image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`.
+
+For more information, please refer
+to [Image Pull Policy](/docs/concepts/containers/images/#image-pull-policy)
+section.
+-->
+当你完成某个镜像的验证时，可以在你的 Pod 清单通过摘要值来指定该镜像，例如：
+`registry-url/image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`。
+
+需要更多信息，请参考[镜像拉取策略](/zh/docs/concepts/containers/images/#image-pull-policy)章节。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:94]:
```suggestion
要了解更多信息，请参考[镜像拉取策略](/zh-cn/docs/concepts/containers/images/#image-pull-policy)章节。
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+resources:
+
+* [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/cosigned)
+* [Configuration Options](https://github.com/sigstore/cosign/tree/main/config)
+-->
+## 使用准入控制器验证镜像签名
+
+有一些非控制平面镜像（例如
+[conformance image](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md)
+），也可以在部署时使用
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:112]:
```suggestion
有一些非控制平面镜像（例如 [conformance 镜像](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md)），
也可以在部署时使用
```

**Comment by Sea-n** [comment]
Hey @ihuibin, as [previously](https://github.com/kubernetes/website/pull/33956#issuecomment-1152831419) said:
We have moved Mandarin contain from `content/zh/docs` to `content/zh-cn/docs`, please rebase and move you file accordingly.

/hold

**Comment by ihuibin** [comment]
Am I create a new verify-signed-images.md patch in `website/content/zh-cn/docs/tasks/administer-cluster/`  ?

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
+-->
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` 用于对以 `KEYLESS` 模式签名的镜像进行验证。想要进一步了解 `KEYLESS`，请参考
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)。
+{{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:60]:
```suggestion
{{< note >}}
<!--
`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
in `KEYLESS` mode. To learn more about keyless signing, please refer to
[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures).
-->
`COSIGN_EXPERIMENTAL=1` 用于对以 `KEYLESS` 模式签名的镜像进行验证。想要进一步了解 `KEYLESS`，请参考
[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)。
{{< /note >}}
```

**Comment by tengqm** [comment]
Place this page under 'website/content/zh-cn/docs/tasks/administer-cluster/' please.

**Comment by Sea-n** [comment]
Hey @ihuibin,

If you need help, please join [Kubernetes Slack](https://slack.k8s.io/).

如果你不知道該怎麼用 git 達到上面需要的更動，麻煩加入 Slack 的 `#kubernetes-docs-zh` 頻道，有很多人可以協助你完成

如同 tengqm 講的，這幾個 PR 已經過有段時間了，我們希望能盡快完成相關轉換任務

**Comment by ihuibin** [comment]
How?Close this PR and create a new one in zh-cn?

**Comment by tengqm** [comment]
/close
Proposing a new PR as replacement.

**Pull request closed without merging**